rgw/v2/tests/s3_swift/test_list_object_versions.py

```diff
@@ -0,0 +1,167 @@
+            log.info("no of buckets to create: %s" % config.bucket_count)
+            for bc in range(config.bucket_count):
+                bucket_name = utils.gen_bucket_name_from_userid(user_name, rand_no=bc)
+                aws_reusable.create_bucket(cli_aws, bucket_name, endpoint)
+                log.info(f"Bucket {bucket_name} created")
+
+                if config.test_ops.get("enable_version", False):
+                    log.info(f"enable bucket versioning on bucket: {bucket_name}")
+                    aws_reusable.put_get_bucket_versioning(
+                        cli_aws, bucket_name, endpoint
+                    )
+
+                if config.test_ops.get("create_object", None) is True:
+                    if config.test_ops.get("large_objects", None) is True:
+                        object_args = list(config.mapped_sizes.items())
+                        with ThreadPoolExecutor() as tp:
+                            futures = [
+                                tp.submit(
+                                    lambda oc, size: (
+                                        setattr(config, "obj_size", size),
+                                        log.info(
+                                            f"uploading object: {utils.gen_s3_object_name(bucket_name, oc)}"
+                                        ),
+                                        put_object_s3client.upload_object_via_s3client(
+                                            s3_client,
+                                            bucket_name,
+                                            utils.gen_s3_object_name(bucket_name, oc),
+                                            TEST_DATA_PATH,
+                                            config,
+                                            user,
+                                        ),
+                                    ),
+                                    oc,
+                                    size,
+                                )
+                                for oc, size in object_args
+                            ]
+                            for future in as_completed(futures):
+                                try:
+                                    future.result()
+                                except Exception as e:
+                                    log.error(f"Object upload failed: {e}")
+                                    raise
+
+    # List all object versions using your reusable method
+    try:
+        version_list = aws_reusable.list_object_versions(cli_aws, bucket_name, endpoint)
+    except ClientError as err:
+        raise AssertionError(f"Failed to perform object version listing: {err}")
+
+    # Check for any crashes during the execution
+    crash_info = reusable.check_for_crash()
+    if crash_info:
+        raise TestExecError("ceph daemon crash found!")
+
+
+if __name__ == "__main__":
+    test_info = AddTestInfo("Listing objects of a bucket via radosgw-admin and boto")
```

Comment:
Fix the info to reflect the test

Reply:
@manisha-reddem , as the listing is included via aws, could you place the config and suite yamls in rgw/v2/tests/aws.